assets/src/edit-story/index.js

```diff
@@ -31,19 +32,25 @@ import './style.css'; // This way the general editor styles are loaded before al
  *
  * @param {string} id       ID of the root element to render the screen in.
  * @param {Object} config   Story editor settings.
+ * @param flags
  */
-const initialize = (id, config) => {
+const initialize = (id, config, flags) => {
   const appElement = document.getElementById(id);
 
   // see http://reactcommunity.org/react-modal/accessibility/
   Modal.setAppElement(appElement);
 
-  render(<App config={config} />, appElement);
+  render(
+    <FlagsProvider features={flags}>
```

Comment:
Should this rather be called inside `<App>`?

Reply:
I was going to ask this question

Reply:
the `flagged` sample code wraps the app completely: https://www.npmjs.com/package/flagged

But I guess it could go inside. I guess it's more in the case that you wanted to flag-guard anything in the App component itself. Thoughts?

Reply:
Main question: what happens when we don't have a `FlagsProvider` at all? Does it default to all flags are off?

Generally, I like this here - this helps with integration tests. Integration tests would likely to start from `app/index.js` and it'd be easy to wrap the whole integration test with the `<FlagsProvider flags={flags-I-am-currently-testing}>`. We could always pass flags around in args, but then we just increase the dependencies on them.

Reply:
If you don't have a FlagsProvider, then the app fails (which is arguably better).

Reply:
Ok. We can still deal with this on integration level, so I'm ok if this is here and not in `app/index.js` and slightly prefer that even.

Reply:
The story editor playground will need to be updated as well:
https://github.com/google/web-stories-wp/blob/master/.storybook/stories/playground/index.js#L55

Reply:
Indeed! Could add some knobs to the storybook to easily toggle flags there